docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Why We Created the Site Showcase
+date: 2018-06-15
+author: Shannon Soper
+tags: [""]
+---
+ 
+We recently finished the first phase of the [Site Showcase](https://next.gatsbyjs.org/showcase/), which is a searchable, filterable library of sites built with Gatsby!
+
+![Featured Sites](featured-sites.png)
+
+![Filtered Sites](filtered-sites.png)
+
+## Submitting your site
+
+If you’d like to submit a site to the showcase, please follow the [instructions in this document](https://github.com/gatsbyjs/gatsby/blob/v2/docs/docs/site-showcase-submissions.md).
+ 
+# Why build a site showcase?
```

Comment:
There should only be one H1 per page, and that's typically the article title. Multiple H1s can screw with search engines.

To avoid that, let's make all headlines one level deeper:

```diff
- # Why build a site showcase?
+ ## Why build a site showcase?
```